agentconfig/agentconfig.go

```diff
@@ -82,6 +82,9 @@ const (
 
 	osConfigPollIntervalDefault = 10
 	osConfigMetadataPollTimeout = 60
+
+	//GDU domain
```

Comment:
please avoid abbreviations, // Default Google API domain will be slightly better